ffmpeg/ffmpeg_test.go

```diff
@@ -1869,3 +1869,53 @@ func TestResolution_Clamp(t *testing.T) {
 	checkError = require.Error
 	test(l, Size{300, 600}, portrait, Size{300, 600})
 }
+
+func TestDurationFPS_GetCodecInfo(t *testing.T) {
+	run, dir := setupTest(t)
+	defer os.RemoveAll(dir)
+
+	//Generate test files
+	cmd := `
+	cp "$1/../data/duplicate-audio-dts.ts" test.ts
+	ffprobe -loglevel warning -show_format test.ts | grep duration=
```

Comment:
please match against the actual duration / frame rate values for these ffprobes as the other testcases do ...

to get the actual values, put an `exit 1` at the end of this script to induce a failure and run the test. the lines matched by grep will be printed, put those in here, then remove the `exit` and hopefully everything passes

Reply:
Added the expected ffprobe duration and fps values to the grep command

Reply:
Is there a way to pass/fail the test based on the values from ffprobe or is this good? https://github.com/livepeer/lpms/pull/407/commits/c85f8176a93bc6e89904d50bc7bcfdc187138740

Reply:
if the grep does not match anything, it will fail (nonzero exit code) so this is good